### PR TITLE
[BAHIR-99] flink-connector-kudu

### PR DESCRIPTION
**Pull request by eskabetxe**
Java library to integrate Apache Kudu and Apache Flink. 
Main goal is to be able to read/write data from/to Kudu using the DataSet and DataStream Flink's APIs.

### CODE REVIEW

**Comment by eskabetxe** [comment]
@lresende could you check this

File: flink-connector-kudu/dockers/role/docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (eskabetxe) [reply flink-connector-kudu/dockers/role/docker-entrypoint.sh]: done

File: flink-connector-kudu/dockers/run_kudu_tests.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (eskabetxe) [reply flink-connector-kudu/dockers/run_kudu_tests.sh]: done

File: flink-connector-kudu/dockers/start-images.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (eskabetxe) [reply flink-connector-kudu/dockers/start-images.sh]: done

File: flink-connector-kudu/dockers/stop-images.sh [hunk 4]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (eskabetxe) [reply flink-connector-kudu/dockers/stop-images.sh]: done

File: pom.xml [hunk 5]
```diff
@@ -274,6 +275,7 @@
               <exclude>**/dependency-reduced-pom.xml</exclude>
               <exclude>**/target/**</exclude>
               <exclude>**/README.md</exclude>
+              <exclude>**/dockers/**</exclude>
```
[the remark this answers is not in the document]
Comment (eskabetxe) [reply pom.xml]: removed

**Comment by meijies** [comment]
@eskabetxe Can't insert the data with `UNIXTIME_MICROS` type.

**Comment by lresende** [comment]
@eskabetxe This has been merged to master, could you please close the PR (as the commit didn't trigger that)

**Pull request closed without merging**